src/Files.Core/Services/ISystemPinService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Files.Core.Storage;
+
+namespace Files.Core.Services
+{
+	/// <summary>
+	/// A service that allows to manage pinning and unpinning items in common OS places.
```

Comment:
Is this specific to the start menu, or does it refer to the sidebar as well?

Reply:
It is the refactored `SecondaryTileHelper` class. The functionality remains unchanged.

Reply:
It's a little generic right now, can we clarify in the name that it's for the start menu?

Reply:
Renamed to `IStartMenuService`